signer/src/storage/postgres.rs

```diff
@@ -401,6 +409,87 @@ impl PgStore {
 
         get_utxo(aggregate_key, sbtc_txs)
     }
+
+    /// Attempts to retrieve an entire transaction package by id.
+    ///
+    /// TODO: This could be made more efficient and shorter by returning a table
+    /// with all the necessary information in one query, and doing a custom
+    /// `FromRow` implementation. I just didn't want to spend more time on it
+    /// now.
+    async fn get_sweep_transaction_by_txid(
+        &self,
+        sweep_txid: &model::BitcoinTxId,
+    ) -> Result<Option<model::SweepTransaction>, Error> {
+        let transaction: Option<model::SweepTransaction> = sqlx::query_as(
+            "
+            SELECT
+                txid,
+                signer_prevout_txid,
+                signer_prevout_output_index,
+                signer_prevout_amount,
+                signer_prevout_script_pubkey,
+                amount,
+                fee,
+                created_at_block_hash,
+                market_fee_rate
+            FROM
+                sweep_transactions
+            WHERE
+                txid = $1;
+        ",
```

Comment:
nit: we should use a consistent style for our queries.

Reply:
Adjusted in [d181170](https://github.com/stacks-network/sbtc/pull/710/commits/d1811705b06b1167b0e178f41a92f81e67242b14)